controllers/kogitoserverlessworkflow_controller.go

```diff
@@ -115,7 +116,7 @@ func buildEnqueueRequestsFromMapFunc(c client.Client, build *operatorapi.KogitoS
 	for i := range list.Items {
 		workflow := &list.Items[i]
 
-		match, err := utils.SameOrMatch(build, workflow)
+		match, err := sameOrMatch(build, workflow)
```

Comment:
where is "match" used in this context?

Reply:
IDK, we will probably refactor this once we get to the prod profile. It was implemented by @davidesalerno. Any thoughts?